### PR TITLE
feat: move claim offline when respondent litigant in person (CMC-1068)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1068

### Change description ###

Move claim to PROCEEDS_WITH_OFFLINE_JOURNEY when respondent is not represented.
Notification sent to applicant solicitor.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by swalker125** [comment]
Email
Screenshot_2020-12-08 Email – Civil Unspecified – GOV UK Notify

**Comment by swalker125** [comment]
Create claim bpmn
Screenshot 2020-12-08 at 16 02 09

**Comment by swalker125** [comment]
New question screen
Screenshot_2020-12-08 HMCTS Manage Cases

**Comment by swalker125** [comment]
Submission screen
Screenshot_2020-12-08 HMCTS Manage Cases(1)

File: e2e/api/steps.js [hunk 1]
```diff
@@ -17,26 +18,40 @@ let caseId, eventName;
...
     });
   },
 
+  createClaimWithRespondentLitigantInPerson: async (user) => {
+    eventName = 'CREATE_CLAIM';
+    caseId = null;
+    caseData = {};
+    await apiRequest.setupTokens(user);
+    await apiRequest.startEvent(eventName);
+    await validateEventPages(data.CREATE_CLAIM_RESPONDENT_LIP);
```
Comment (pd-dudkiewicz) [inline e2e/api/steps.js:39]: We could use the same data as we do for regular claim creation and only change `respondent1Represented` value before submission. Might be a bit simpler and easier to see that different state depends only on that one value. Thoughts?

**Review by pd-dudkiewicz — DISMISSED** [review]

File: e2e/api/steps.js (hunk 1 above)
Comment (swalker125) [reply e2e/api/steps.js:39]: I've raised this tech debt @pd-dudkiewicz https://tools.hmcts.net/jira/browse/CMC-1109 you happy for this to be solved there?

**Review by sabahirfan — APPROVED** [review]

**Review by pd-dudkiewicz — APPROVED** [review]

**Pull request merged**